source/_integrations/http.markdown

```diff
@@ -117,7 +117,7 @@ The [Set up encryption using Let's Encrypt](/blog/2015/12/13/setup-encryption-us
 
 ## APIs
 
-On top of the `http` integration is a [REST API](/developers/rest_api/), [Python API](/developers/python_api/) and [WebSocket API](/developers/websocket_api/) available. There is also support for [Server-sent events](/developers/server_sent_events/).
+On top of the `http` integration is a [REST API](/developers/rest_api/) and [WebSocket API](/developers/websocket_api/) available.
```

Comment:
We should consider to point directly to the docs which are now part of the Developers's documentation.

- https://developers.home-assistant.io/docs/api/rest
- https://developers.home-assistant.io/docs/api/websocket

Reply:
I've changed that in #14774